e2e/wdio/core/tests/grid-list.e2e-spec.ts

```diff
@@ -0,0 +1,122 @@
+import { GridListPo } from '../pages/grid-list.po';
+import {
+    click, dragAndDrop, elementArray,
+    getAttributeByName,
+    getCSSPropertyByName,
+    getElementArrayLength, getText,
+    refreshPage,
+    scrollIntoView, waitForClickable, waitForElDisplayed
+} from '../../driver/wdio';
+
+import { text, productTitle, textLocked, warningColor, successColor, neutralColor, errorColor, color, backGroundColor,
+    fontWeight, bold, classAttribute, isSelected } from '../fixtures/appData/grid-list-content';
+
+describe('Grid-list test suite', function() {
```

Comment:
please use arrow function

Reply:
This was discussed with QA team. We use arrow functions only for steps ('it')